### ISSUE TITLE
Fix Request: false positive in AvoidTimeUnitConfusion

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#666]
```
class Issue666 {
    @Autowired
    public RetrieveCache(final boolean flagTimeToLive) { // false pos
    }
    public void confuse(Boolean useExpiryTime) { // good
    }
}
```
